tests/helpers/format.test.ts

```diff
@@ -19,10 +18,34 @@ describe("formatValue on number", () => {
     expect(formatValue(0.00000000001)).toBe("0");
     expect(formatValue(0.000000000001)).toBe("0");
 
+    expect(formatValue(-1)).toBe("-1");
+    expect(formatValue(-0.1)).toBe("-0.1");
+    expect(formatValue(-0.01)).toBe("-0.01");
+    expect(formatValue(-0.001)).toBe("-0.001");
+    expect(formatValue(-0.0001)).toBe("-0.0001");
+    expect(formatValue(-0.00001)).toBe("-0.00001");
+    expect(formatValue(-0.000001)).toBe("-0.000001");
+    expect(formatValue(-0.0000001)).toBe("-0.0000001");
+    expect(formatValue(-0.00000001)).toBe("-0.00000001");
+    expect(formatValue(-0.000000001)).toBe("-0.000000001");
+    expect(formatValue(-0.0000000001)).toBe("-0");
+    expect(formatValue(-0.00000000001)).toBe("-0");
+    expect(formatValue(-0.000000000001)).toBe("-0");
+
+    expect(formatValue(10.123456789)).toBe("10.123456789");
+    expect(formatValue(100.123456789)).toBe("100.12345679");
+    expect(formatValue(1000.123456789)).toBe("1000.1234568");
+    expect(formatValue(10000.123456789)).toBe("10000.123457");
+    expect(formatValue(-10000.123456789)).toBe("-10000.12346");
```

Comment:
I wonder if we really want a difference in the number of digits between the positive case and the negative case.

Reply:
My reasoning was "display as much digits as possible, while keeping the number less than 13 characters so it fit nicely in a default cell width with default font size". So there it make sense to display one mroe digit if we don't have the minus sign.

But we could display always one less digit, leaving a space for the minus sign. I don't have a strong opinion on this.

Reply:
Removed a digit to always leave a space for the minus sign